### PR TITLE
Added ADMS Characteristics classes and circle support to building

### PR DESCRIPTION
**Pull request by Hilbrand**
And removed some obsolete classes/methods.

### CODE REVIEW

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/FileFormat.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright the State of the Netherlands
+ * Crown copyright
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/FileFormat.java:3]: Not sure if we have a license check here, but is this double copyright statement OK?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/Building.java [hunk 2]
```diff
@@ -53,4 +57,15 @@ public void setHeight(final double height) {
     this.height = height;
   }
 
+  public double getRadius() {
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/Building.java:60]: Probably need to add this (and related changes) to IMAER as well. Not saying it should be done in this PR, just an observation.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ADMSSourceCharacteristics.java [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+  private double volumetricFlowRate;
+
+  private double specificHeatCapacity;
+
+  private double percentNOxAsNO2;
+
+  private double fm;
+
+  private double fb;
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ADMSSourceCharacteristics.java:54]: What are these?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/EffluxType.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  EffluxType(final int type, final boolean actualOrNtp, final boolean velocity, final boolean volumeFlux, final boolean rho, final boolean fmbm,
+      final boolean massFlux) {
+    this.type = type;
+    this.actualOrNtp = actualOrNtp;
+    this.velocity = velocity;
+    this.volumeFlux = volumeFlux;
+    this.rho = rho;
+    this.fmbm = fmbm;
+    this.massFlux = massFlux;
+  }
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/EffluxType.java:45]: Not sure what this enum determines or what each boolean represents (would guess expected fields?), but is this related to output velocity? Might be we're only supporting velocity with actual outflow velocity values being specified if I understood last wednesdays discussion right.

Also, is efflux the same as outflow?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/EffluxType.java [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.shared.domain.v2.characteristics.adms;
+
+/**
+ * Efflux type determines
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/EffluxType.java:20]: determines...?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/SourceCharacteristics.java [hunk 6]
```diff
@@ -23,8 +24,11 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
 
+import nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSSourceCharacteristics;
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/SourceCharacteristics.java:27]: Not sure if package cycles are a problem, but this import does introduce one.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ReleaseAtNTP.java [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.shared.domain.v2.characteristics.adms;
+
+/**
+ * Emission parameters given at actual temperature and pressure or normal temperature and pressure (1atm, 273.15K): 0=actual,1=NTP
+ */
+public enum ReleaseAtNTP {
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ReleaseAtNTP.java:22]: Seems a bit weird name for this enum, since one of the possible values is NTP.
What does NTP stand for? `Normal Temperature and Pressure?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/EffluxType.java (hunk 4 above)
Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/EffluxType.java:45]: I've updated the comment. This enum specifies which of the fields are available for the specific efflux type. This is not yet used, but intended for among others the ui to guide what input fields to display.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/SourceCharacteristics.java (hunk 6 above)
Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/SourceCharacteristics.java:27]: I'll move the class to avoid cycle.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ReleaseAtNTP.java (hunk 7 above)
Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ReleaseAtNTP.java:22]: The name was taken from the `.apl` file configuration, but agreed it's not very clear. Will change it to: `ReleaseTemperatureAndPressure`.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/Building.java (hunk 2 above)
Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/Building.java:60]: Yes this needs to be added.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/ADMSSourceCharacteristics.java [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+  private static final long serialVersionUID = 1L;
+
+  private String buildingId;
+
+  private BuoyancyType buoyancyType;
+
+  private EffluxType effluxType;
+
+  private ReleaseTemperatureAndPressure releaseAtNTP;
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/ADMSSourceCharacteristics.java:36]: change in enum name perhaps warrants a change in property name?
